package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
```

Comment:
* According to the task you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.